logr.go

```diff
@@ -281,6 +286,22 @@ func (l Logger) Info(msg string, keysAndValues ...any) {
 	}
 }
 
+// Debug is a convenience function. It's equivalent to
+//
+//	V(LevelDebug).Info
+func (l Logger) Debug(msg string, keysAndValues ...any) {
```

Comment:
This function might not be useful enough to justify its own existence, in particular when there's then two ways to do the same thing. Linters also would need to be adapted, so... :-1:  ?